packages/wp-now/src/run-cli.ts

```diff
@@ -42,6 +42,11 @@ export async function runCli() {
 					type: 'string',
 					default: DEFAULT_PHP_VERSION,
 				});
+				yargs.option('wp', {
+					describe: 'WordPress version to use.',
```

Comment:
it would be very helpful to leave an example here. there's the perpetual question: "does it need a `v` before the number?" and we could answer that by showing: "WordPress version to use: e.g. '--wp 6.2'"

Reply:
Love the idea. Added here https://github.com/WordPress/wordpress-playground/pull/252/commits/425bf291b13767bb6b5ebe20b993017d4a363e3f